servicex/transformer_manager.py

```diff
@@ -155,6 +155,7 @@ def create_job_object(request_id, image, chunk_size, rabbitmq_uri, workers,
             metadata=client.V1ObjectMeta(labels={'app': "transformer-" + request_id}),
             spec=client.V1PodSpec(
                 restart_policy="Always",
+                priority_class_name=current_app.config.get('TRANSFORMER_PRIORITY_CLASS'),
```

Comment:
We like to keep a working app.conf template in `app.conf.template` - please add this there.